WiFiPasswordRetriever.py

```
@@ -1,4 +1,5 @@
 import subprocess
+import numpy
```

Ruff (0.8.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)